tests/utils.go

```diff
@@ -2871,14 +2871,10 @@ func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter
 		shouldConfigureIpv6Batch := append([]expect.Batcher{
 			&expect.BSnd{S: "\n"},
 			&expect.BExp{R: prompt},
-			&expect.BSnd{S: "ip a | grep -q eth0\n"},
-			&expect.BExp{R: prompt},
-			&expect.BSnd{S: "echo $?\n"},
-			&expect.BExp{R: "0"},
-			&expect.BSnd{S: "ip address show dev eth0 scope global | grep -q inet6\n"},
-			&expect.BExp{R: prompt},
-			&expect.BSnd{S: "echo $?\n"},
-			&expect.BExp{R: "1"}})
+			&expect.BSnd{S: "ip a | grep -q eth0; echo $?\n"},
+			&expect.BExp{R: "\n0"},
```

Comment:
instead of \n0 
(?m)^0$
this would also work, if preferred ?

it means multi line mode (which is needed, else it looks at the current buffer that it already filled)
and then it searches for a line which has only 0 on it.

Reply:
I think I understand now. Maybe adding "$" will be even more accurate. `"\n0$"`

Reply:
it means end of buffer if `?m` isnt used
end of buffer is reached easily, at the moment you have just those 2 characters "at hand", 
its not meaning its indeed come with end of line afterwards
this is why i suggested the above regex